src/Feature/Account/code/Controllers/AuthenticationController.cs

```diff
@@ -14,73 +14,54 @@
 
 namespace Wooli.Feature.Account.Controllers
 {
+    using System;
     using System.Web.Mvc;
     using System.Web.Security;
 
+    using Foundation.Account.Models.Authentication;
     using Foundation.Account.Services.Authentication;
-    using Foundation.Commerce.Context;
+    using Foundation.Base.Controllers;
     using Foundation.Commerce.Models.Authentication;
-    using Foundation.Commerce.Models.Entities.Users;
-    using Foundation.Commerce.Providers;
-    using Foundation.Commerce.Services.Cart;
-    using Foundation.Commerce.Services.Tracking;
     using Foundation.Extensions.Extensions;
 
-    public class AuthenticationController : Controller
+    using Sitecore.Diagnostics;
+
+    public class AuthenticationController : BaseController
     {
         private readonly IAuthenticationService authenticationService;
 
-        private readonly ICartService cartService;
-
-        private readonly ICommerceTrackingService commerceTrackingService;
-
-        private readonly ICustomerProvider customerProvider;
-
-        private readonly IVisitorContext visitorContext;
-
         public AuthenticationController(
-            IAuthenticationService authenticationService,
-            ICartService cartService,
-            ICommerceTrackingService commerceTrackingService,
-            ICustomerProvider customerProvider,
-            IVisitorContext visitorContext)
+            IAuthenticationService authenticationService)
         {
+            Assert.ArgumentNotNull(authenticationService, nameof(authenticationService));
+
             this.authenticationService = authenticationService;
-            this.cartService = cartService;
-            this.customerProvider = customerProvider;
-            this.commerceTrackingService = commerceTrackingService;
-            this.visitorContext = visitorContext;
         }
 
+        // TODO: Add error transfer to FE
```

Comment:
- `Login(LoginRequest request \{email, password}\)`
- change only routing on FE (login/logout)
- add attribute validation in login request model